Snippets/Snippets_5/Monitoring/EndpointConfig.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace MonitoringNotifications.ServiceControl.Notifications
+{
+    using System;
+    using NServiceBus;
+    using NServiceBus.Persistence;
+
+    #region MessageFailedEndpointConfig 5.0
+    public class EndpointConfig : IConfigureThisEndpoint, AsA_Server
```

Comment:
It's a minor pitfall, if overlooked the whole thing will fail... 
Happy to remove it if you feel strongly about it...